docs/modules/reference/pages/telemetryd/protocols/bmp/bmp.adoc

```diff
@@ -26,6 +26,8 @@ View metrics and statistics related to that peer and apply thresholding.
 * *BMP Peer Status Adapter:* Handles BMP Peer Up and Down messages that the BMP parser receives and parses, and converts them to OpenNMS events.
 Allows alarm creation based on up/down status.
 
-The parser and adapters are configurable in the telemetryd-configuration.xml file.
+You can configure the parser and adapters in the `telemetryd-configuration.xml` file.
 
 TIP: The https://github.com/OpenNMS-forge/bmp-playground[BMP Playground] provides a virtual lab that you can use to simulate BGP and BMP, and to see how the {page-component-title} components can be configured at runtime.
+
+//Do all the adapters need to be enabled? If not, what are the minimum ones required?
```

Comment:
Depending on the use case, if they want to know the status of Peers, BMP Peer Status Adapter can be used.  

if they want to know all the BGP updates, BGP Persisting adapter can be used